packages/portal-proto/src/features/proteinpaint/OncoMatrixWrapper.tsx

```diff
@@ -56,7 +56,7 @@ export const OncoMatrixWrapper: FC<PpProps> = (props: PpProps) => {
       setPickedCases(cases);
       setShowCreateCohort(true);
     },
-    [createSet, pickedCases],
+    [],
```

Comment:
@craigrbarnes  Just wanted to bring to your attention that linter was complaining here for unnecessary deps. Please verify.

Reply:
Changes are fine as both dependencies are functions from useState, which will be stable. This discussion seems to conclude it can be done either way:
https://github.com/reactjs/react.dev/issues/4506